browser/ui/views/sidebar/sidebar_items_contents_view.cc

```diff
@@ -15,6 +15,8 @@
 #include "base/strings/string_util.h"
 #include "base/strings/utf_string_conversions.h"
 #include "brave/app/vector_icons/vector_icons.h"
+#include "brave/browser/brave_browser_process.h"
+#include "brave/browser/misc_metrics/process_misc_metrics.h"
```

Comment:
Why do we need to include these two lines for all builds, I mean with and without `ENABLE_AI_CHAT`?